docs/source/specs/openapi.json

```diff
@@ -1,8 +1,8 @@
 {
     "openapi": "3.0.0",
     "info": {
-        "description": "The API for Cost Management.  You can find out more about     Cost Management at [https://github.com/project-koku/](https://github.com/project-koku/).",
-        "version": "1.0.0",
+        "description": "The API for Project Koku and OpenShift cost management. You can find out more about Project Koku at [https://github.com/project-koku/](https://github.com/project-koku/).",
+        "version": "0.9.0",
```

Comment:
Version should be higher not lower here.

Reply:
Normally, I believe we used 1.0.0 because it was the default, but we are not GA and we are planning on changing the API, so defining this as 1.0.0 is misleading and could cause problems in the future.

Reply:
I am ok with 1.0.0, I just don't want customers opening bugs because we change the API from 1.0.0 to 1.0.1

Reply:
@adberglund what do you think?

I agree with Luke that this is a problem, but it would allow us to update the minor number every time we have a minor update.

The way I see it:
Version 1.0.0 would be consistent with the endpoint and will not downgrade existing version. Problem: using semantic versioning that would mean we are new minor versions of this API will not break the current implementation and we know that is wrong.

Version 0.9.0 would not be consistent with the endpoint /v1/ and will downgrade the existing version. However, it would allow us to refer to the api as beta until we are ready to release it as 1.0.0, meanwhile we should update the API version to 0.9.x, 0.10.x

This would only be done until GA of the API. Once we promote to version 1 we would need a wider strategy, as there should be two endpoints, the production endpoint and the beta endpoint, with different versioning. Version 2 would only be needed once we need a major update on the API.

So it would be:
- Endpoint /v1/ --> For production. Stable branch
- Endpoint /v1beta/ --> For production beta. Beta branch, it is expected some bugs that will break the API but that should be fixed before moving the API to /v1/

Reply:
After our discussion I am happy to go with 1.0.1 for now.